v0.3.1/functions/Knative/logs-handler-function/logs-handler-function.yaml

```diff
@@ -15,4 +15,8 @@ spec:
       url: "https://github.com/OpenFunction/samples.git"
       sourceSubPath: "v0.3.1/functions/Knative/logs-handler-function/"
   serving:
-    runtime: "Knative"
\ No newline at end of file
+    runtime: "Knative"
+    template:
+      containers:
+        - name: function
+          imagePullPolicy: Always
```

Comment:
Always is used with : latest tag, this can be omitted if there is a non-latest tag for the image

Reply:
Maybe we can change pull policy to IfNotPresent and add comment out this section for non latest cases

Reply:
ok